src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -624,4 +624,13 @@
   <data name="Global_using_directives" xml:space="preserve">
     <value>Global 'using' directives</value>
   </data>
+  <data name="Convert_0_to_record" xml:space="preserve">
+    <value>Convert '{0}' to record</value>
```

Comment:
nit:

Do we need to lock `record` localization?

Reply:
Addressed in follow-up PR